README.md

```diff
@@ -10,43 +10,35 @@ Migration consists of an analysis part and a code modification part.
 
 Next to the standalone CLI command an add'tl API is provided, which simplifies integration into other tools (e.g. into custom node.js scripts, grunt).
 
-IMNPORTANT: The modified source code needs to be manually reviewed and tested. There is no 100% guarantee that the modified code works as expected.
+**IMNPORTANT**: The modified source code needs to be manually reviewed and tested. There is no 100% guarantee that the modified code works as expected.
 
 **If you want to migrate your codebase, please see the [migration guide](./docs/guide/migrationguide.md)**
 
 
 ## Installation
 
-Run the following command to delete the previous installation to avoid conflicts
-```cli
-npm uninstall ui5-migration --global
-```
+### Requirements
+- [Node.js](https://nodejs.org/) (**version 8.5 or higher** ⚠️)
 
 The migration tool is currently available for experimental usage. It can be installed with npm.
 ```cli
-npm install ui5-migration --global
+npm install --global @ui5/migration
```

Comment:
We might also consider to explain the linking for the time being as long as we did not release the ui5-migration tool to NPM

Reply:
I would opt for releasing it as quickly as possible and leaving the explanation here meanwhile

Reply:
maybe we could do this in a separate PR (then it is separate and we could simply "revert" once it is published.